exaudfclient/base/javacontainer/script_options/converter.h

```diff
@@ -15,33 +15,32 @@ namespace SWIGVMContainers {
 
 namespace JavaScriptOptions {
 
-class ScriptOptionsConverter {
+class Converter {
 
     public:
-        ScriptOptionsConverter(std::function<void(const std::string &msg)> throwException,
-                               std::vector<std::string>& jvmOptions);
-
+        Converter();
     
-        void getExternalJarPaths(std::string & src_scriptCode);
+        void convertExternalJar(const std::string & value);
+
+        void convertScriptClassName(const std::string & value);
 
-        void getScriptClassName(std::string & src_scriptCode);
+        void convertJvmOption(const std::string & value);
 
-        void convertImportScripts(std::string & src_scriptCode);
+        const std::set<std::string> & getJarPaths() const {
```

Comment:
Hm, you use a move for the Jvm options, but not here, is there a reason.

Reply:
The JavaVMImpl class iterates over all items of this set in the constructor, anyway. So there is no need to move the set. On the other hand, if you say that we should have a common interface for the return values, maybe then it makes sense...